Riot/Modules/Room/NotificationSettings/SwiftUI/FormSectionHeaderView.swift

```diff
@@ -0,0 +1,48 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct FormSectionHeaderView: View {
+    
+    @Environment(\.theme) var theme: Theme
+    var text: String
+    
+    var body: some View {
+        Text(text)
+            .foregroundColor(Color(theme.textSecondaryColor))
+            .padding(.top)
+            .padding(.leading)
+            .padding(.bottom, 8)
+            .font(Font(theme.fonts.footnote))
```

Comment:
Is `.footnote` the right size for a header? Looks pretty small (but I don't know where to look for a comparison so might just be me).

Reply:
It's what is used in the designs yea. I can check if it is a different size to the usual iOS section header and check with design.

Reply:
Yea looks like headers are usually 15pt and footnote is 13pt. Will confirm with design and change it to `subheadline`